src/DataLayer/helpers/tests/filters.spec.ts

```diff
@@ -1,85 +1,34 @@
 // (C) 2007-2018 GoodData Corporation
 import { AFM } from '@gooddata/typings';
-import {
-    Granularities
-} from '../../constants/granularities';
-import {
-    isNotEmptyFilter,
-    mergeFilters
-} from '../filters';
-
-describe('isNotEmptyFilter', () => {
-    it('should return true for date filter', () => {
-        const dateFilter: AFM.IRelativeDateFilter = {
-            relativeDateFilter: {
-                dataSet: {
-                    identifier: 'date filter'
-                },
-                from: 0,
-                to: 0,
-                granularity: Granularities.MONTH
-            }
-        };
-
-        expect(isNotEmptyFilter(dateFilter)).toBe(true);
-    });
-
-    it('should return false for empty positive filter', () => {
-        const attributeFilter: AFM.IPositiveAttributeFilter = {
-            positiveAttributeFilter: {
-                displayForm: {
-                    identifier: 'empty filter'
-                },
-                in: []
-            }
-        };
-
-        expect(isNotEmptyFilter(attributeFilter)).toBe(false);
-    });
-
-    it('should return true for filled positive filter', () => {
-        const attributeFilter: AFM.IPositiveAttributeFilter = {
-            positiveAttributeFilter: {
-                displayForm: {
-                    identifier: 'filled filter'
-                },
-                in: ['1', '2', '3']
-            }
-        };
-
-        expect(isNotEmptyFilter(attributeFilter)).toBe(true);
-    });
-
-    it('should return false for empty negative filter', () => {
-        const attributeFilter: AFM.INegativeAttributeFilter = {
-            negativeAttributeFilter: {
-                displayForm: {
-                    identifier: 'empty filter'
-                },
-                notIn: []
-            }
-        };
-
-        expect(isNotEmptyFilter(attributeFilter)).toBe(false);
-    });
-
-    it('should return true for filled negative filter', () => {
-        const attributeFilter: AFM.INegativeAttributeFilter = {
-            negativeAttributeFilter: {
-                displayForm: {
-                    identifier: 'filled filter'
-                },
-                notIn: ['1', '2', '3']
-            }
-        };
-
-        expect(isNotEmptyFilter(attributeFilter)).toBe(true);
+import { isEmptyFilter, mergeFilters } from '../filters';
+import * as input from './filters.input.fixtures';
+
+//
```

Comment:
empty comments :)

Reply:
i use these to visually delimit blocks

Reply:
its not something which we are used to do it... for structuring tests we are using `describe` 

just saying that somebody after you it probably removes because its not our typical way how to delimit code block :)

Reply:
i've removed it although i suggest we all learn from current design trends and use more whitespace :)